package.json

```diff
@@ -15,7 +15,7 @@
   "license": "AGPL-3.0",
   "dependencies": {
     "@edx/edx-bootstrap": "^1.0.0",
-    "@edx/paragon": "3.1.2",
+    "@edx/paragon": "3.4.7",
```

Comment:
I do have one final detail to follow-up on this PR which relates to some propType errors I expected, but didn't see. They were stemming from changes in edx/paragon I thought were necessary, but which we also ran into trouble trying to test locally. Following up on that in just a little bit! (It seems like the proptype validations went away when I bumped the paragon version, but my understanding was that I should still expect them even if we were up-to-date with `@edx/paragon`'s `master`...

Reply:
It looks like proptype validations aren't running against paragon components in the new version we're upgrading into. FYI @fysheets, since this may be a larger issue. We may have a number of proptype issues masked by this issue.

I haven't dug in super far, but I expect this may also be the result of how we use post-build paragon code from within studio-frontend.

Reply:
We're moving forward from this with a bumped paragon version, which should suffice to eliminate any proptype validation errors.